internal/diag/diag.go

```diff
@@ -105,12 +105,15 @@ func logCommand(ctx context.Context, cfg conf.Config, args args.Config) {
 	envConfig, isEnv := cfg.(conf.EnvironmentConfig)
 
 	prefix := "<no prefix matched in config>"
+	strip := "<no strip matched in config>"
```

Comment:
I feel the text here doesn't quite make sense, as only the prefix option is used during matching.  Could we please use an empty string instead?